src/lit-html.ts

```diff
@@ -416,6 +416,7 @@ export class NodePart implements SinglePart {
       this._previousValue = undefined;
     } else if (partIndex < itemParts.length) {
       const lastPart = itemParts[partIndex - 1];
+      itemParts.length = partIndex;
```

Comment:
🎉 the first time I have ever seen someone set length on an `Array` is today 🎂

Reply:
first time for everything!

I added a comment